src/autocomplete/mysql/grammar/MySqlParser.g4

```diff
@@ -587,9 +587,9 @@ alterSpecification
     | ADD COLUMN? LR_BRACKET uid columnDefinition (COMMA uid columnDefinition)* RR_BRACKET                                            # alterByAddColumns
     | ADD indexFormat = (INDEX | KEY) uid? indexType? indexColumnNames indexOption*                                                   # alterByAddIndex
     | ADD (CONSTRAINT name = uid?)? PRIMARY KEY index = uid? indexType? indexColumnNames indexOption*                                 # alterByAddPrimaryKey
-    | ADD (CONSTRAINT name = uid?)? UNIQUE indexFormat = (INDEX | KEY)? indexName = uid? indexType? indexColumnNames indexOption*     # alterByAddUniqueKey
+    | ADD (CONSTRAINT name = uid?)? UNIQUE indexFormat = (INDEX | KEY)? uid? indexType? indexColumnNames indexOption*     # alterByAddUniqueKey
     | ADD keyType = (FULLTEXT | SPATIAL) indexFormat = (INDEX | KEY)? uid? indexColumnNames indexOption*                              # alterByAddSpecialIndex
-    | ADD (CONSTRAINT name = uid?)? FOREIGN KEY indexName = uid? indexColumnNames referenceDefinition                                 # alterByAddForeignKey
```

Comment:
There was a naming clash, because i've defined `indexName` rule below. I don't see the point in this inline alias, so I've decided to remove it